### ISSUE TITLE
Invalid GEDCOM data passed to Fact::_construct

### ISSUE THREAD

**Issue by peterjanes** [fisharebest/webtrees#3898]
Webtrees version: 2.0.16
PHP version: 7.4.15

I tried to import a GEDCOM file generated by Family Tree Maker (22.0.0.1410) and it failed to import a "Fact" related to a recently-updated media item.

Here's the general form of the record:

```
0 @M269@ OBJE^M
1 FILE C:\...\birthday.jpg^M
2 TITL 1- Person - some information about them 1
2- Person - some information about them 2
3- Person - some information about them 3
4 - Person - some information about them 4
...
10 - Person - some information about them 10
11 - Person - some information about them 11
12 - Person - some information about them 12
13 - Person - some information about them 13
14 - Person - some information about them 14^M
```

...and of the error message:

```
Invalid GEDCOM data passed to Fact::_construct(1 0 - Person - some information about them 10,M269)
```

I had to remove all of the lines from `10 - Person ...` to the end of the record to get it to import successfully. I believe it may be a limitation on the length of the record text, but I haven't looked much further into the code than the constructor and Gedcom::REGEX_TAG constant.

**Comment by fisharebest** [comment]
It looks like you have entered multi-line data (i.e. containing line breaks) into the `TITL` field, and Family Tree Maker has exported the line-breaks directly into the GEDCOM file.

In other words, the value of the line `2 TITL ...` spills over to the subsequent lines.

This is invalid.  Lines at level 1 and higher must have the format `[level] [space] [tag] [space] [value]`

Multi-line values must use the `CONT` tag for each new line.  Since we are continuing from level 2, these would be `3 CONT ...`

The `TITL` tag does not actually support multi-line text -  but if it did, then multi-line data would look like this:

```
0 @M269@ OBJE
1 FILE C:\...\birthday.jpg
2 TITL 1- Person - some information about them 1
3 CONT 2- Person - some information about them 2
3 CONT 3- Person - some information about them 3
3 CONT 4- Person - some information about them 4
...
```

TL;DR - this GEDCOM file is corrupt/invalid, and it would appear to be a bug in FTM.
